main/opcodes.zkasm

```diff
@@ -2828,10 +2834,13 @@ opCREATE2:
     E                   :MSTORE(lastMemOffset)
     C                   :MSTORE(lastMemLength)
                         :CALL(saveMem)
-
+    C => E
     CTX                 :MSTORE(originAuxCTX)
     $ => B              :MLOAD(txDestAddr)
     GAS - 32000 => GAS  :JMPN(outOfGas)
+    ; Cost to hash the initialisation code before
+    ${(C+31)/32} => C
+    GAS - 6*C => GAS    :JMPN(outOfGas)
```

Comment:
This division should be done with DivArith
Also the substraction of the gas it might need a LT comprobation first, as is it done in `saveMemGAS` in order to avoid the JMPN "problem" that only checks the first 64 bits